cmd/res/profiles/camera.yaml

```diff
@@ -360,7 +360,7 @@ deviceResources:
     properties:
       valueType: "Object"
       readWrite: "R"
-  - name: "GetConfigurations"
+  - name: "Configurations"
```

Comment:
I think I kind of agree with @ChristianDarr-personal here, as the prefix could help avoid confusion. The thing you have to remember is that the onvif APIs have separate scoped endpoints, so `GetConfigurations` is calling `GetConfigurations` of the `ptz` scope, so that is why the name does not include it. In EdgeX we do not have a concept of scope, all commands are in the same area. The command `DeviceCapabilities` is calling the `GetCapabilities` of the `device` scope. Similarily, the `MediaCapabilities` calls `GetCapabilities` of the `media` scope. without that distinction, the name would be clashing.

Reply:
All the other ONVIF configuration related functions have the scope names included in them such as `GetMetadataConfiguration`, `GetVideoEncoderConfiguration`, etc.. except this. I left it from consistency perspective. I can change into `PTZConfigurations` if that's going to help, the others I don't think there is a need. Let me know.